pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -940,3 +940,86 @@ eval range from 0 to 4m step 1m sort_desc(test_metric)
   test_metric{case="histogram with NaN"}  {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}}
   test_metric{case="histogram with +Inf"} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}}
   test_metric{case="histogram with -Inf"} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}}
+
+clear
+
+# absent() tests
+load 6m
+  series{case="-"} _x11
+  series{case="0a"} _x11
+  series{case="0b"} _x11
+  series{case="0"} 0x11
+  series{case="a"} _ 1 _ 1 _ 1 _ 1 _ 1 _ 
+  series{case="b"} _ _ 2 _ 2 _ 2 _ 2 _ 2 
+  series{case="c"} _ _ _ 3 _ _ _ 3 _ _ _ 
+
+eval instant at 0m absent(series)
+  # expected no result
+
+eval instant at 0m absent(series > Inf)
+	{} 1
+
+eval instant at 0m absent(series{case="-"})
+  # expected no result
+
+eval instant at 0m absent(series{case="a"})
+  # expected no result
+
+eval instant at 8m absent(series{case="a"})
+  # expected no result
+
+eval instant at 0m absent(series{case=~"(a|b)"})
+	{} 1
+
+eval instant at 8m absent(series{case=~"(a|b)"})
+  # expected no result
+
+eval range from 0s to 40m step 4m absent(series)
+  # expected no result
+
+eval range from 0s to 40m step 4m absent(series{case="-"})
+	{case="-"} 1x10
+
+eval range from 0s to 40m step 4m absent(series{case="0"})
+  # expected no result
+
+eval range from 0s to 40m step 4m series{case=~"(a|b)"}
+  series{case="a"} _ _ 1 _ _ 1 _ _ 1 
+  series{case="b"} _ _ _ 2 2 _ 2 2 _ 2 2 
```

Comment:
This should be this, no?

```suggestion
  {} 1 1 _ _ _ _ _ _ _ _ _
```

Reply:
This is just the series range evaluation `series{case=~"(a|b)"}`. The absent evaluation itself `absent(series{case=~"(a|b)"})` is in the next line. It is easier to see the expected output of absent after seeing the multiple series range evaluation above.

Reply:
To make this less confusing, let's just remove that range query evaluation c9de184e2b07e39a9014e3962b3789f122dfd048